test/unit/env/env.js

```diff
@@ -56,6 +56,16 @@ class EnvTest extends BaseTest {
     process.env.NODE_ENV = 'production'
     t.true(Env.isProduction())
   }
+
+  async is (t) {
+    process.env.NODE_ENV = 'testing'
+    t.false(Env.is('production'))
+    t.true(Env.is('testing'))
+
+    process.env.NODE_ENV = 'production'
+    t.true(Env.is('production'))
+    t.false(Env.is('testing'))
+  }

```

Comment:
Can you please add a test where `process.env.NODE_ENV = ''` or `process.env.NODE_ENV = undefined` and test that `Env.is('local')` is false?